src/editor/VideoViewer.js

```diff
@@ -0,0 +1,244 @@
+/*
+ * Copyright (c) 2013 - present Adobe Systems Incorporated. All rights reserved.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a
+ * copy of this software and associated documentation files (the "Software"),
+ * to deal in the Software without restriction, including without limitation
+ * the rights to use, copy, modify, merge, publish, distribute, sublicense,
+ * and/or sell copies of the Software, and to permit persons to whom the
+ * Software is furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+ * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+ * DEALINGS IN THE SOFTWARE.
+ *
+ */
+
+define(function (require, exports, module) {
+    "use strict";
+
+    var DocumentManager     = require("document/DocumentManager"),
+        VideoViewTemplate   = require("text!htmlContent/video-view.html"),
+        ProjectManager      = require("project/ProjectManager"),
+        PreferencesManager  = require("preferences/PreferencesManager"),
+        LanguageManager     = require("language/LanguageManager"),
+        MainViewFactory     = require("view/MainViewFactory"),
+        Strings             = require("strings"),
+        Filer               = require("filesystem/impls/filer/BracketsFiler"),
+        Path                = Filer.Path,
+        StringUtils         = require("utils/StringUtils"),
+        FileSystem          = require("filesystem/FileSystem"),
+        UrlCache            = require("filesystem/impls/filer/UrlCache"),
+        FileUtils           = require("file/FileUtils"),
+        Content             = require("filesystem/impls/filer/lib/Content"),
```

Comment:
You're failing travis, because this should be lower-case 'content` vs. `Content`:

```
{ Error: Error: ENOENT: no such file or directory, open '/home/travis/build/mozilla/brackets/src/filesystem/impls/filer/lib/Content.js'
```